pages/agent/v3/hooks.md

```diff
@@ -1,102 +1,111 @@
 # Buildkite Agent hooks
 
-Hooks extend or override the built-in behaviours of the Buildkite agent and
-Bootstrap binaries.
+Hooks allow you extend or override the built-in behaviour of the Buildkite Agent as it runs jobs in a variety of phases using a variety of programming languages.
```

Comment:
allow **you** to?

Also, I think s/behaviour/behavior/ would match the US English style guide of Buildkite Docs? That was already there though.